keyboards/bear_face/v2/keymaps/default/keymap.c

```diff
@@ -82,8 +68,25 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 */
 };
 
-//macros to allow the user to set whatever default layer they want, even after reboot
 
+// Caps Lock LED indicator
+void keyboard_pre_init_kb(void) {
+    //Sets LED pin as output
+    gpio_set_pin_output(F7);
+
+    keyboard_pre_init_user();
+};
+
+bool led_update_kb(led_t led_state) {
+    // Caps Lock LED indicator toggling code here
+    bool res = led_update_user(led_state);
+    if(res) {
+        gpio_write_pin(F7, led_state.caps_lock);
+    }
+    return res;
+};
```

Comment:
Keyboard level code at the keymap level is also incorrect.

However this would be better implemented as <https://docs.qmk.fm/reference_info_json#indicators>.

Reply:
Must have missed this after putting it in `keyboard.json` - fixed